packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -1203,11 +1203,18 @@ export class DashboardModel {
                 `${DashboardsTableName}.dashboard_id`,
                 `${DashboardVersionsTableName}.dashboard_id`,
             )
-            .where(`${DashboardsTableName}.dashboard_uuid`, dashboardUuid);
-
+            .whereIn(`${DashboardsTableName}.dashboard_uuid`, dashboardUuids)
+            .where(
```

Comment:
I believe this filter by last version was missing, and it was getting wrong results, since it was checking the chart across all versions.

Reply:
@rephus I think we need to add a comment to the top of this method.
Check this PR: https://github.com/lightdash/lightdash/pull/11016
> backend will only delete orphans if, and only if, they do not belong to any tile. This means that version reverting will now work for charts created within the dashboard, even if they get removed from the tile in the next dashboard version save.